src/playground/service.clj

```diff
@@ -136,6 +136,8 @@
                     components))
    :name  ::context-injector})
 
+(pedestal-component/using-component [:db])
```

Comment:
this does nothing, delete

In general try to clean up such things before opening PRs